### ISSUE TITLE
Distillation towers and post MV oil

### ISSUE THREAD

**Issue by 0lafe** [GTNewHorizons/GT-New-Horizons-Modpack#4378]
#### Which modpack version are you using?
2.0.7
#

So distillation towers are essentially single block distilleries that have a better fuel efficiency for oil distillation. 50L oil in a distillation towers will give all products 50L would give in an MV single block distillery, but for 1/4 the oil cost. In this way DTs prioritize oil efficiency over power efficiency, as they will use 4x less oil for a given set of outputs compared to single blocks. It will also essentially function 2x faster than LV efficiency if you value each output equally, as it's the time for one distillation at MV, for 4 outputs, instead of 4 processes at 2x the speed. 

However, the distillation tower overclock normally, so running an HV distillation tower (the tier you unlock it) is the same power efficiency as running 4 sets of 4 LV distilleries. You'll get the same output rates from both, for the same power investment, however the DT will eat 4x less oil. By EV, the tower is actually 2x worse than using single blocks, 4x worse by IV. 

The bonus of 4x less oil isn't a huge bonus by HV+ as the oil pump will supply more oil than you could hope to use. So by EV, it's not really worth it to use the tower, and EV is just one tier past the creation of the multi. 

My suggestion is to adjust the recipe for oils/Nat gas that the tower uses to function on speeds related to a perfectly overclocked HV setup as opposed to a standard MV single block. This way they'll need to be fed voltages of the tier you make it in, and you don't start off overclocking it. This would be done by either increasing the outputs by 8x the current rate, or reducing the time by 8x, or something in between. The eu/t also would need to be changed to 384eu/t. This pushes distillation towers to be the dominant distillation method pre LuV, being on par with 4 EV arrays at IV. I'd also suggest raising the input fluid amount to only make them 2x better with fuel efficiency. Would mean you need to pump more when using distillation tower setups, but as pumping is already much faster than usage, it just raises pumping eu cost closer to an amp of the pumps voltage tier. 

PA distilleries kinda beat this out later on in LuV, but you would need 16 which is sort of insane, but still somewhat useful depending on your setup.

**Comment by Bluebine** [comment]
Oil's already really damn OP. I don't get why you're trying to nerf everything else, yet you're somehow OK with buffing what's already such a strong power gen option?

**Comment by sthegreat** [comment]
I disagree with this as it is essentially just a clean buff to oil. While currently there is a choice between distillery and distill tower, and it would make sense for distillation tower to be a more clear upgrade, this has the side effect of massively improving the speed of everyone's cetane setups. I also don't like the idea of increasing the oil input of the distill tower, seems weird when it's usually same input.

**Comment by 0lafe** [comment]
>Oil's already really damn OP. I don't get why you're trying to nerf everything else, yet you're somehow OK with buffing what's already such a strong power gen option?

Doesn't buff oil, it buffs distillation towers. Everything they'd be able to do with this change, small distilleries can already do, and do better. Problem is the distillation tower is a direct downgrade from singleblocks. They're just singleblock distilleries that have 4x fuel eff. They're HV, but there's very little incentive to power them past even EV, as small distilleries will start to overtake them exponential at EV. This being 1 power tier past the tier of creation for it, which is weird as EV could be a starting power for the tower. All this does is make the tower slightly more inline with single block speeds. The bonus from the multi is fuel efficiency, which isn't really applicable to base oil distillation, as you're limited by power/speed way before oil production rates are the issue. It's still amazing for most other distillation options, but just kinda bad for oil (by oil I mean oil alone, not the oil line. Normal/light/heavy/medium is not great when distilled in a tower)

>I disagree with this as it is essentially just a clean buff to oil. While currently there is a choice between distillery and distill tower, and it would make sense for distillation tower to be a more clear upgrade, this has the side effect of massively improving the speed of everyone's cetane setups. I also don't like the idea of increasing the oil input of the distill tower, seems weird when it's usually same input.

Technically it just buffs the DT as the ability to do these rates with the same power req is doable with singles already. For comparison, an EV distillation tower vs single block array for diesel has outputs of 40keu/t for the array and 6keu/t for the tower. Maybe 8x is too much, but at least a 2x would probably be a good idea. Even a 4x makes them worse than singles. 
The fuel efficiency stuff is great in general, just doesn't really pertain to oil distillation, so the 4x bonus there is essentially useless and only really acts as a 4x life span increase for pumped oil. 

>massively improving the speed of everyone's cetane setups.

Only if the only slow part of the setup is the tower itself for base oil distillation, which could also be improved without this change by simply using singleblock distilleries.

**Comment by MMunier** [comment]
This is still an issue and should probably be addressed, by giving the DT efficient overclocks or something similar.
Whilst the overall performance DTs affects diesel powergen -- it also affects a whole lot else.
In general the DT is the bottleneck for almost any setup dependant on it, like `Acidic Osmium Solution` the platline just to name an example.

This basically forces players to either spam singleblock distilleries (prpbably not great for avoiding uneccesary performance hits)  or using a MDT for almost any setup.

**Comment by MMunier** [comment]
The MegaDT is fine, I was just arguing that they were basically neccesary everywhere because regular DT's are just underwhelming, which is basically this issue.

**Comment by 0lafe** [comment]
Sure, but shouldn't they be kind of underwhelming by those tiers? If you give the mega DT better OCs, you would be able to do more recipes with the same amount of them. Which seems like a more tier appropriate solution than buffing an HV machine for UV+ usage

**Comment by bombcar** [comment]
Right now the only thing *really* disguising how 0lafe-OP diesel is is that everyone rushes DT right away, it's slow, and is kinda underwhelmed.

Buffing the DT to be in line with the singles (which I think is actually *reasonable*) would likely reveal this immediately.

**Comment by Savallator** [comment]
Imho the singleblock distilleries should just be nerfed.... Make every Distillery recipe 4x slower, and give the DT a 4x speed bonus (or preferably 4 paralells), and the problem is fixed. Then singles are good early game, but the distillation tower is just better.

Look at the Distilled water recipes: The problem was actually noticed, and solved by giving distilled water 2 seperate recipes, one slow for the singleblock and one with 50x the amount in the same time for the DT.

**Comment by Glease** [comment]
4x slower make it sucks at early game. Quick question: how much EU of light fuel can a distillery make per tick? If you cannot answer this question off the top of your head you better go do some math first before make any suggestion.

**Comment by Savallator** [comment]
It does not make it suck early game, fuel (and diesel) is just hilariously OP in its current state.
A LV Distillery will make 375 EU/t, while consuming 24.
After the nerf it would make 94, which is still some profit. Of course this does not amount for the desulfurization. But that could just be solved by decreasing the power consumption of the recipe. You can make it consume only 12 for example.

**Comment by Savallator** [comment]
Imho we should come to a conclusion on how the DT should perform compared to singleblocks, and then early game oil recipes can be adjusted accordingly.

**Comment by Dream-Master** [comment]
Use bc oil. Underground oil is fine like it is.

**Comment by Savallator** [comment]
Imho we should split this discussion into a DT rebalance and a oil rebalance. Of course the DT rebalance has to be decided first, and then oil can be tuned accordingly.

**Comment by 0lafe** [comment]
Imo the DT is fine as it is. The main issue I see is that oil is so easy obtainable that you can use single block distilleries instead of the DT. When oil is a limiting factor, the DT will always be better. I think reducing pumping speed of underground oil would once again make the DT better.

A side effect would be that diesel couldn't compare to things like CBD and HOG

**Comment by leagris** [comment]
If you want con consider slower process for single-block distilery, take care to check distilleries from Bluidcraft and from Forestry Still. Gregtech started by having the very same operation with a slightly better output of 30% instead of 25%.

Now I advise against nerfing distilleries.

Think positively and give the Distillation Tower a good speed and efficiency gain with its own distinct recipes map.

In real-life small distillation setups are impaired by a smaller heat gradient that requires active cooling (water, or special cooler and or compressor and heat exchanger). The tiny gradient makes it only suitable for a fixed temperature distillation output (single output).

On the other hand, Distillation Towers benefit from a very large (tall) gradient. Natural convection and taller designs greatly reduces the cost or need for active cooling. Passive dissipation costs nothing. This is the reason why Distillation Towers in Gregtech should be like 2× more energetically efficient and probably 8× faster (for same energy tier). A DT should be able to replace at least 8 distilleries.

**Comment by Savallator** [comment]
Yeah, okay, that could work. But i would advise against giving DT's perfect OC, like someone said, because that would obsolete the Mega DT. i suspect the Mega DT is often forgotten, while it is actually quite cheap once you reach late HV/early IV, and very energy efficient due to paralell (effectively similar to perfect oc).

**Comment by bombcar** [comment]
I'll admit I didn't know the MEGA DT wasn't UHV tier or something

**Comment by Savallator** [comment]
it just needs a ton of stainless and 64 normal DT's. Mega DT and mega freezer are actually cheap, just the Mega EBF is more expensive due to coils and mufflers needed. Technically all megas could be build in HV.

**Comment by Glease** [comment]
Mega DT can be more realistically built during EV~IV not HV. Anyway, reopening this for now.

**Comment by bombcar** [comment]
Here is sulfuric light in a LV distillery:

![image](https://user-images.githubusercontent.com/533931/148583519-d5432324-1fd2-44e0-9351-f8735e3aa4de.png)

25 "Oil" -> 25 "Sulfuric Light Fuel" for 480 EU and one second.

compared to in a DT:

![image](https://user-images.githubusercontent.com/533931/148583592-e8ef19ae-0584-4b2d-824b-bd4d2216a3bb.png)

50 "Oil" -> 50 "Sulfuric Light Fuel" for 1920 EU and one second. Sure you get byproducts, but at what cost 😅 

For the same EU in single blocks (LV spam) you would convert 100 Oil to 100 sulfuric light fuel.

Also that recipe seems a bit nuts as it gives you 50 input for 20+15+50+60 = 145 output .... 

Perhaps it should be dropped to be the same speed as singles but split into 4 parts that add up instead of magically making more ...

**Comment by bombcar** [comment]
A simple solution that would put the DT on "par" with the single at MV would be to make that recipe be 100L to 30/100/40/60.

**Comment by Savallator** [comment]
This is not only an oil problem. Look at diluted acid recycling, platline stuff etc... The distillation tower is almost everywhere not really worth it if you go strictly for efficiency. It is rather worth it to go straight to the mega then in Midgame. For distilled water apparently there is a manual fix for it by manipulating the recipe (the dt recipe is far better, as it should be ). But do we really want to change every recipe, or rather give the DT a flat buff and then look at oil stuff after that.

**Comment by bombcar** [comment]
Yeah in a perfect world we'd define the DT recipes for various things and it would autogenerate the appropriate single-block recipes (with some being off if needed, etc). I doubt that's how it's done

**Comment by 0lafe** [comment]
I'd recommend closing this issue. I opened another one relating to slowing down the rigs which fixes the issue addressed in my OP. Buffing the DT with other processes should be another ticket probably
